<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectWindow</class>
 <widget class="QMainWindow" name="ConnectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConnectWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbHost">
      <property name="text">
       <string>Hoszt és Port:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="lbColon">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbService">
      <property name="text">
       <string>SSID:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="lbConnections">
      <property name="text">
       <string>Mentett kapcsolatok:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" rowspan="7">
     <widget class="QListWidget" name="lwConnections"/>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="pbSave">
      <property name="text">
       <string>Mentés</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+S</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLineEdit" name="lEUsername">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="3">
     <widget class="QPushButton" name="pbConnect">
      <property name="text">
       <string>Kapcsolódás</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="lbPassword">
      <property name="text">
       <string>Jelszó:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lbUsername">
      <property name="text">
       <string>Felhasználónév:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lEPort"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="lEHost"/>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QLineEdit" name="lEPassword">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QLineEdit" name="lEService"/>
    </item>
    <item row="6" column="3">
     <widget class="QRadioButton" name="rbSavePassword">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>&amp;Jelszó mentése</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lEHost</tabstop>
  <tabstop>lEPort</tabstop>
  <tabstop>lEService</tabstop>
  <tabstop>lEUsername</tabstop>
  <tabstop>lEPassword</tabstop>
  <tabstop>pbConnect</tabstop>
  <tabstop>pbSave</tabstop>
  <tabstop>rbSavePassword</tabstop>
  <tabstop>lwConnections</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
